### ISSUE TITLE
Why is "pip install tensorflowjs" installing tf-nightly-2.0-preview? It messes my environment up and keras stops working without using tf.keras. 

### ISSUE THREAD

**Issue by rachita97** [tensorflow/tfjs#1534]
To get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

#### TensorFlow.js version
tensorflowjs   1.0.1 

#### Browser version
Mozilla Firefox 66.0.3 (64-bit)
#### Describe the problem or feature request
I am trying to train a keras model and then convert it to tensorflowjs. I already have a converted model but when I try to predict something it just stops there and is doing nothing. I've read that I have to convert it before I save it and also run it in keras, not in tf.keras. The problem I think comes from tensorflowjs being dependent on tf-nightly-2.0-preview. 

pip show tensorflowjs
Name: tensorflowjs
Version: 1.0.1
Summary: Python Libraries and Tools for TensorFlow.js
Home-page: https://js.tensorflow.org/
Author: Google LLC
Author-email: opensource@google.com
License: Apache 2.0
Location: /home/razvan/anaconda3/envs/python3.6/lib/python3.6/site-packages
Requires: numpy, six, keras, h5py, tensorflow-hub, tf-nightly-2.0-preview
Required-by: 

#### Code to reproduce the bug / link to feature request
import os
import numpy as np
import tensorflow
KERAS_BACKEND=tensorflow //i had to do this because otherwise would run on theano //backend by default
import keras
import pandas as pd
from keras import backend as K
from keras.layers.core import Dense, Activation, Dropout
from keras.optimizers import Adam
from keras.metrics import categorical_crossentropy
from keras.preprocessing.image import ImageDataGenerator
from keras.preprocessing import image
from keras.models import Model
from keras.applications import imagenet_utils
from keras.callbacks import EarlyStopping, ReduceLROnPlateau, ModelCheckpoint
from sklearn.metrics import confusion_matrix
from sklearn.model_selection import train_test_split
import shutil
import itertools
import matplotlib.pyplot as plt
%matplotlib inline

mobile = keras.applications.mobilenet.MobileNet()

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-8-88ac919df462> in <module>
      1 # create a copy of a mobilenet model
----> 2 mobile = keras.applications.mobilenet.MobileNet()

~/anaconda3/envs/python3.6/lib/python3.6/site-packages/keras/applications/__init__.py in wrapper(*args, **kwargs)
     26             kwargs['models'] = models
     27             kwargs['utils'] = utils
---> 28         return base_fun(*args, **kwargs)
     29 
     30     return wrapper

~/anaconda3/envs/python3.6/lib/python3.6/site-packages/keras/applications/mobilenet.py in MobileNet(*args, **kwargs)
      9 @keras_modules_injection
     10 def MobileNet(*args, **kwargs):
---> 11     return mobilenet.MobileNet(*args, **kwargs)
     12 
     13 

~/anaconda3/envs/python3.6/lib/python3.6/site-packages/keras_applications/mobilenet.py in MobileNet(input_shape, alpha, depth_multiplier, dropout, include_top, weights, input_tensor, pooling, classes, **kwargs)
    231 
    232     if input_tensor is None:
--> 233         img_input = layers.Input(shape=input_shape)
    234     else:
    235         if not backend.is_keras_tensor(input_tensor):

~/anaconda3/envs/python3.6/lib/python3.6/site-packages/keras/engine/input_layer.py in Input(shape, batch_shape, name, dtype, sparse, tensor)
    176                              name=name, dtype=dtype,
    177                              sparse=sparse,
--> 178                              input_tensor=tensor)
    179     # Return tensor including _keras_shape and _keras_history.
    180     # Note that in this case train_output and test_output are the same pointer.

~/anaconda3/envs/python3.6/lib/python3.6/site-packages/keras/legacy/interfaces.py in wrapper(*args, **kwargs)
     89                 warnings.warn('Update your `' + object_name + '` call to the ' +
     90                               'Keras 2 API: ' + signature, stacklevel=2)
---> 91             return func(*args, **kwargs)
     92         wrapper._original_function = func
     93         return wrapper

~/anaconda3/envs/python3.6/lib/python3.6/site-packages/keras/engine/input_layer.py in __init__(self, input_shape, batch_size, batch_input_shape, dtype, input_tensor, sparse, name)
     37         if not name:
     38             prefix = 'input'
---> 39             name = prefix + '_' + str(K.get_uid(prefix))
     40         super(InputLayer, self).__init__(dtype=dtype, name=name)
     41 

~/anaconda3/envs/python3.6/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py in get_uid(prefix)
     72     """
     73     global _GRAPH_UID_DICTS
---> 74     graph = tf.get_default_graph()
     75     if graph not in _GRAPH_UID_DICTS:
     76         _GRAPH_UID_DICTS[graph] = defaultdict(int)

AttributeError: module 'tensorflow' has no attribute 'get_default_graph'

**Comment by tafsiri** [comment]
cc @pyu10055

**Comment by caisq** [comment]
@rachita97 It is recommended to install tensorflowjs in a separate pipenv or virtualenv to avoid conflicting with user's existing tensorflow/keras installations.

**Comment by pyu10055** [comment]
this issue is a duplicate of #https://github.com/tensorflow/tfjs/issues/1546
